### PR TITLE
PT-159405636 Add channel ws contract methods docs

### PR DESCRIPTION
**Pull request by cytadela8**
Documentation for aeternity/aeternity#2239

This also adds docs for `channel.update.new_contract` and `channel.update.call_contract` methods.

### CODE REVIEW

**Comment by lucafavatella** [comment]
I am taking the liberty of labelling as WIP as not planned for this release - so to ease filtering of open PRs.

File: node/api/channel_ws_api.md [hunk 1]
```diff
@@ -335,6 +335,97 @@ Roles:
...
+    "deposit": 10,
+    "vm_version": 3
+  },
+  "tag": "new_contract"
+}```
```
Comment (lucafavatella) [inline node/api/channel_ws_api.md]:
```suggestion
}
```
```

Comment (lucafavatella) [inline node/api/channel_ws_api.md]: New line required for markup.

File: node/api/channel_ws_api.md [hunk 2]
```diff
@@ -335,6 +335,97 @@ Roles:
 }
 ```
 
+## Contracts
```
Comment (lucafavatella) [inline node/api/channel_ws_api.md]: How does this fit in the table of contents above in this document?

**Comment by uwiger** [comment]
LGTM, besides the changes requested by @lucafavatella

**Pull request merged**